tasks/create-internal-request/create-internal-request.yaml

```diff
@@ -53,6 +53,46 @@ spec:
     - name: input
       description: Workspace to store the params and responses for the internalRequest
   steps:
+    - name: merge-internal-request-parameters
+      image:
+        quay.io/hacbs-release/release-utils@sha256:5733ece907aa70d6ebced36484c936731e8b27bfcf87fed226a0ecf95047a6b8
+      script: |
+          #!/usr/bin/env bash
+          DATA_FILE="$(workspaces.input.path)/$(params.dataPath)"
+          if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No data JSON was provided."
+            exit 1
+          fi
+
+          IR_PARAMS_FILE="$(workspaces.input.path)/$(params.subdirectory)/ir-params-$$.txt"
```

Comment:
Why do we need the `$$`? The subdirectory is only used by this pipeline run, so it can be simply `ir-params.txt`, no?

Reply:
The fbc-pipeline uses this task multiple times in the same workspace, so it was causing the next tasks to use wrong input. The idea is moving out from the `create-internal-request` task. So this won't be needed soon.

Reply:
I see, ok, fine then.